src/extensions/default/bramble/main.js

```diff
@@ -326,7 +265,7 @@ define(function (require, exports, module) {
                     fn: data.fn,
                     value: typeof value !== "object" ? value : undefined
                 }), "*");
-            });
+            }, false);
```

Comment:
I feel like you might have explained this to me before. Doesn't this parameter default to `false` if it isn't specified?